plugins/vanillicon/class.vanillicon.plugin.php

```diff
@@ -114,7 +117,7 @@ function userPhotoDefaultUrl($user, $options = []) {
         if (!$email) {
             $email = val('UserID', $user, 100);
         }
-        $hash = md5($email);
+        $hash = substr(sha1($email), 0, VanilliconPlugin::TRUNCATED_HASH_LENGTH);
```

Comment:
Can you make this change in the Gravatar plugin too?

Reply:
This is all set. I also hoped to update the version used to Vanillicon v2, but Gravatar doesn't play nicely with SVGs.